Client.Examples/Client.Examples.csproj

```diff
@@ -13,7 +13,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Configuration.FileExtensions" Version="3.1.19" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="5.0.2" />
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="5.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="3.1.19" />
```

Comment:
Hm why this was dropped to a lower version ? :thinking:

Reply:
There were two completely different versions of Microsoft.Extensions.Logging here. I also found a lower version in Client.IntegrationTests.csproj. We should avoid multiple versions, so this consolidates Client.IntegrationTests.csproj from 3.1.14 to 3.1.19 while consolidating the 5.x version to 3.1.19. I would suggest moving both projects to 5.x when you move to 5.x (or really .NET 6)

Reply:
> Thanks @Christian-Oleson
> 
> could you please adjust your commit messages and follow the contribution guide https://github.com/camunda-cloud/zeebe/blob/develop/CONTRIBUTING.md#commit-message-guidelines

I think I fixed the commit message? Let me know if I'm misunderstanding anything.